packages/ember-metal/lib/meta.js

```diff
@@ -78,6 +87,9 @@ export function Meta(obj, parentMeta) {
   this._chains = undefined;
   this._tag = undefined;
 
+  // initial value for all flags right now is false
```

Comment:
it help future travelers could we also make a comment here describing the layout of this bitfield?